<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="open_page">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>130</y>
          <width>361</width>
          <height>101</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>초간단 영화추천기</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="start_btn">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>260</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>시작</string>
        </property>
       </widget>
       <widget class="QPushButton" name="quit_btn">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>300</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>종료</string>
        </property>
       </widget>
       <widget class="QPushButton" name="made_by">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>340</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>만든 사람</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-9</y>
          <width>811</width>
          <height>611</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>img/qvfz_736u_210906.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>290</y>
          <width>331</width>
          <height>361</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../../../Downloads/화면_캡처_2023-06-29_161330-removebg-preview.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>400</y>
          <width>211</width>
          <height>161</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>img/images-removebg-preview.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>450</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>자자 영화추천이 공짜</string>
        </property>
       </widget>
       <zorder>label_2</zorder>
       <zorder>label</zorder>
       <zorder>start_btn</zorder>
       <zorder>quit_btn</zorder>
       <zorder>made_by</zorder>
       <zorder>label_3</zorder>
       <zorder>label_4</zorder>
       <zorder>label_5</zorder>
      </widget>
      <widget class="QWidget" name="main_page_1">
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>231</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>재미있게 본 영화를 선택하세요!</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>-50</y>
          <width>351</width>
          <height>191</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>img/rupi.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QWidget" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>791</width>
          <height>441</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(218, 218, 218);</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>40</y>
          <width>111</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>확인하기</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>10</y>
          <width>181</width>
          <height>111</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>img/bubble_2.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <zorder>label_6</zorder>
       <zorder>label_8</zorder>
       <zorder>label_9</zorder>
       <zorder>widget</zorder>
       <zorder>pushButton</zorder>
      </widget>
      <widget class="QWidget" name="main_page_2">
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>20</y>
          <width>221</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="text">
         <string>추천결과</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="contributer_page">
       <widget class="QLabel" name="background_lab">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>811</width>
          <height>581</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>img/heart_img.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>110</y>
          <width>201</width>
          <height>271</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>32</pointsize>
         </font>
        </property>
        <property name="text">
         <string>만든 사람

박소연</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
